packages/core/src/storages/request_queue_v2.ts

```diff
@@ -361,7 +430,8 @@ export class RequestQueue extends RequestProvider {
 
```

Comment:
I cannot comment it below but during code review, I see that we are removing locks one by one in `_clearPossibleLock`.
see https://github.com/apify/crawlee/blob/092d60c1027389d56cf7348b797898648680a95c/packages/core/src/storages/request_queue_v2.ts#L446

There is 200 rps rate limit. I would remove lock in some batches maybe 10 to speed it up.

Reply:
I see. However, I still doubt that there will be any measurable benefit - this code is only executed on migration and there shouldn't be more than ~25 requests in the queue head.